massgov/mayflower
# DP-23669 Add "language bar" links for pages that have  professional translations

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description


## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-23669)

## Steps to Test


1. Visit the Inline Links component on this branch and ensure that it matches the mockup on the Jira ticket: https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-23669_add-language-bar/?p=organisms-inline-links

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.
<img width="1307" alt="j-r-capture 2022-01-14 a la(s) 12 42 20" src="https://user-images.githubusercontent.com/141685/149543187-e44ff5d7-5c7f-4d64-a8db-3ae6a9f82152.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* Inline Links

#### @TODO


*

#### Today I learned...
